pages/map.js

```diff
@@ -43,14 +45,30 @@ export async function getStaticProps() {
     logger.error(e, "ERROR loading tags");
   }
 
+  try {
+    data.events = await getEvents();
+  } catch (e) {
+    logger.error(e, "ERROR loading Events");
+  }
+
+  data.events=data.events.filter((event)=>{
+    // console.log(event.location)
+    return event.location &&
+    event.location.lat && 
+    event.location.lon &&
+    event.location.lat!=null &&
+    event.location.lon!=null
```

Comment:
I think the filter could be done in the DB query, maybe a flag to the `getEvents` function